### PR TITLE
Fix Search Live Tests

### PR DESCRIPTION
**Pull request by khmic5**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

File: sdk/maps/Azure.Maps.Search/tests/SearchClientTestEnvironment.cs [hunk 1]
```diff
@@ -15,6 +15,6 @@ public class SearchClientTestEnvironment : TestEnvironment
                 return endpoint != null ? new Uri(endpoint) : null;
             }
         }
-        public string MapAccountClientId => GetRecordedVariable("AZMAPS_CLIENT_ID");
+        public string MapAccountClientId => GetRecordedVariable("MAPS_CLIENT_ID");
```
Comment (weshaggard) [inline sdk/maps/Azure.Maps.Search/tests/SearchClientTestEnvironment.cs:18]: Should this match the name of the output comming from the test-resources.json file? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/maps/test-resources.json#L44 Similar question for the ENDPOINT_URL above.

Comment (benbp) [reply sdk/maps/Azure.Maps.Search/tests/SearchClientTestEnvironment.cs:18]: The new value is correct, as it follows the `<service directory>_CLIENT_ID` convention for env vars that gets set by the deploy script, not the ARM template.

**Review by benbp — APPROVED** [review]

**Pull request merged**